Services/Utilities/classes/class.ilUtil.php

```diff
@@ -3585,55 +3585,13 @@ public static function deducibleSize($a_mime)
 
 
 	/**
-	* http redirect to other script
-	*
-	* @param	string		$a_script		target script
-	* @static
-	* 
-	*/
+	 * @param $a_script
+	 *
+	 * @deprecated User $DIC->ctrl()->redirectToURL() instead
```

Comment:
Please add some rule for dicto sometime.

Reply:
Good point, we will! @d3r1w will provide some rules in September (along with the rules for File-Operations)